docs/design/mui-best-practices.md

```diff
@@ -39,4 +39,4 @@ If you still decide to build it yourself after contacting us, try and define the
 -  A MUI mixin (a freeform bundle of reuseable styling rules defined inside a theme)
 -  A new [reuseable component](https://mui.com/material-ui/customization/how-to-customize/#2-reusable-component)
 
-Some of the above options require you to extend our MUI theme. See the MUI documentation on [theme composition](https://mui.com/system/styles/advanced/#theme-nesting)
+Some of the above options require you to extend our MUI theme. See the MUI documentation on [theme composition](https://mui.com/material-ui/customization/theming/#nesting-the-theme).
```

Comment:
We were linking a legacy part of the docs toward `@mui/styles` (soon to be deprecated)